features/old/applications/providers/bulk_operations/send_emails.feature

```diff
@@ -22,32 +22,27 @@ Feature: Mass email bulk operations
     Given current domain is the admin domain of provider "foo.3scale.localhost"
       And I don't care about application keys
 
-  Scenario: Emails can be sent without body
+  Scenario: Emails can't be sent without body
     Given I am logged in as provider "foo.3scale.localhost"
     And I am on the applications admin page
     And a clear email queue
     When I check select for "BobApp"
     And I press "Send email"
-    And I fill in "Subject" with "Nothing to say"
-    And I press "Send" and I confirm dialog box within colorbox
```

Comment:
I think you will still need to confirm the dialog

Reply:
No, because the required filed will not allow to open that , confirm dialog will open only after fields are filled.